src/main/java/edu/arizona/kra/irb/onlinereview/CustomProtocolOnlineReviewServiceImpl.java

```diff
@@ -0,0 +1,76 @@
+package edu.arizona.kra.irb.onlinereview;
+
+import org.apache.commons.lang.StringUtils;
+import org.kuali.kra.irb.Protocol;
+import org.kuali.kra.irb.ProtocolDocument;
+import org.kuali.kra.irb.actions.submit.ProtocolSubmission;
+import org.kuali.kra.irb.onlinereview.ProtocolOnlineReviewService;
+import org.kuali.kra.irb.onlinereview.ProtocolOnlineReviewServiceImpl;
+import org.kuali.kra.protocol.ProtocolBase;
+import org.kuali.kra.protocol.ProtocolOnlineReviewDocumentBase;
+import org.kuali.kra.protocol.actions.submit.ProtocolSubmissionBase;
+import org.kuali.kra.protocol.onlinereview.ProtocolOnlineReviewBase;
+import org.kuali.rice.kew.api.exception.WorkflowException;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+import java.util.List;
+
+/**
+ * Created by nataliac on 9/15/17.
+ */
+public class CustomProtocolOnlineReviewServiceImpl extends ProtocolOnlineReviewServiceImpl implements ProtocolOnlineReviewService {
+    private static final Logger LOG = LoggerFactory.getLogger(CustomProtocolOnlineReviewServiceImpl.class);
+
+    public boolean isProtocolInStateToBeReviewed(ProtocolBase protocol) {
+        boolean isReviewable = false;
+        ProtocolSubmission submission = ((Protocol)protocol).getProtocolSubmission();
+        if(submission != null) {
+            try {
+                isReviewable = StringUtils.isNotEmpty(submission.getScheduleId()) || "2".equals(submission.getProtocolReviewTypeCode()) || "7".equalsIgnoreCase(submission.getProtocolReviewTypeCode()) && "112".equalsIgnoreCase(submission.getProtocolSubmissionType().getSubmissionTypeCode());
```

Comment:
Tons of magic strings, I'd suggest maybe spring props for this to keep it out of the code and easier to change; failing that, maybe made into constants for easy spotting at the top of the class; failing that, perhaps comments calling out what they are.

Reply:
Yeah, just took the code from the parent service.. didn't bother switching those...